src/shared/components/mutationTable/column/CosmicColumnFormatter.spec.jsx

```diff
@@ -1,21 +1,132 @@
+        {
+            count: 2,
+            keyword: "TP53 R273 missense",
+            proteinChange: "R273H"
+        },
+        {
+            count: 3,
+            keyword: "TP53 R273 missense",
+            proteinChange: "R273C"
+        },
+        {
+            count: 5,
+            keyword: "TP53 R273 missense",
+            proteinChange: "R273L"
+        },
+        {
+            count: 7,
+            keyword: "PIK3CA R38 missense",
+            proteinChange: "R38H"
+        },
+        {
+            count: 11,
+            keyword: "PIK3CA R38 missense",
+            proteinChange: "R38C"
+        },
+        {
+            count: 13,
+            keyword: "PIK3CA R38 missense",
+            proteinChange: "R38S"
+        }
+    ];
 
-    });
+    const mutation273 = {
+        gene: {
+            hugoGeneSymbol: "TP53"
+        },
+        proteinChange: "R273S",
+        keyword: "TP53 R273 missense"
+    };
 
-    after(()=>{
+    const mutation38 = {
+        gene: {
+            hugoGeneSymbol: "PIK3CA"
+        },
+        proteinChange: "R38S",
+        keyword: "PIK3CA R38 missense"
+    };
+
+    const mutation666 = {
+        gene: {
+            hugoGeneSymbol: "DIABLO"
+        },
+        proteinChange: "V666E",
+        keyword: "DIABLO"
+    };
+
+    const tableData = [[mutation273], [mutation38], [mutation666]];
+
+    let component273, component38, component666;
+
+    before(() => {
+        const columnProps = {
+            cosmicData: keywordToCosmic(cosmicMutations)
+        };
+
+        let data = {
+            name: "Cosmic",
+            tableData: tableData,
+            rowData: [mutation273]
+        };
+
+        // mount a single cell component (Td)
+        component273 = mount(CosmicColumnFormatter.renderFunction(data, columnProps));
+
+        data = {
+            name: "Cosmic",
+            tableData: tableData,
+            rowData: [mutation38]
+        };
 
+        // mount a single cell component (Td)
+        component38 = mount(CosmicColumnFormatter.renderFunction(data, columnProps));
+
+        data = {
+            name: "Cosmic",
+            tableData: tableData,
+            rowData: [mutation666]
+        };
+
+        // mount a single cell component (Td)
+        component666 = mount(CosmicColumnFormatter.renderFunction(data, columnProps));
+    });
+
+    it('component tooltip', () => {
+        assert.isTrue(component273.find('DefaultTooltip').exists(),
+            'Tooltip should exists for TP53 R273 missense mutation');
+        assert.isTrue(component38.find('DefaultTooltip').exists(),
+            'Tooltip should exists for PIK3CA R38 missense mutation');
+        assert.isFalse(component666.find('DefaultTooltip').exists(),
+            'Tooltip should not exist for DIABLO mutation');
     });
 
-    it('what does it do?', ()=>{
+    it('component display value', () => {
+        assert.isTrue(component273.find(`span`).text().indexOf((2 + 3 + 5).toString()) > -1,
```

Comment:
@alisman this is actually a string comparison, not a numerical value comparison. Something like this could be better: 
```JavaScript
...text().contains((2 + 3 + 5).toString())
```